go-controller/pkg/ovn/address_set.go

```diff
@@ -48,12 +59,9 @@ func NewOvnAddressSetFactory() AddressSetFactory {
 	return &ovnAddressSetFactory{}
 }
 
-// ovnAddressSetFactory implements the AddressSetFactory interface
-var _ AddressSetFactory = &ovnAddressSetFactory{}
```

Comment:
Isn't `ovnAddressSetFactory` implementing `AddressSetFactory` interface anymore?

Reply:
It does but we don't have to explicitly define it this way. I removed it to address Dave's comment.

Reply:
sorry, missed Dave's comment before. This is considered good style in golang, and it can be left in. (Likewise the other ones that got removed)

Reply:
Thanks @danwinship . Done the changes.